controllers/games.js

```diff
@@ -1,32 +1,41 @@
 const { selectCategories, selectReviews, selectReviewById,
-  selectCommentsByReviewId } = require('../models/games.js');
-    
-  exports.getCategories = (req, res) => {
-    selectCategories().then((categories) => {
-      res.status(200).send({ "categories": categories });
-    });
-  };
+  selectCommentsByReviewId, insertCommentByReviewId } = require('../models/games.js');
 
-  exports.getReviews = (req, res) => {
-    selectReviews().then((reviews) => {
-      res.status(200).send({ "reviews": reviews });
-    });
-  };
-  
-  exports.getReviewById = (req, res, next) => {
-    selectReviewById(req.params.review_id).then((review) => {
-      res.status(200).send({ "review": review });
-    })
+exports.getCategories = (req, res) => {
+  selectCategories().then((categories) => {
+    res.status(200).send({ "categories": categories });
+  });
+};
+
+exports.getReviews = (req, res) => {
+  selectReviews().then((reviews) => {
+    res.status(200).send({ "reviews": reviews });
+  });
+};
+
+exports.getReviewById = (req, res, next) => {
+  selectReviewById(req.params.review_id).then((review) => {
+    res.status(200).send({ "review": review });
+  })
     .catch((err) => {
       next(err)
     })
-  };
+};
 
-  exports.getCommentsByReviewId = (req, res, next) => {
-    selectCommentsByReviewId(req.params.review_id).then((comments) => {
-      res.status(200).send({ "comments": comments });
-    })
+exports.getCommentsByReviewId = (req, res, next) => {
+  selectCommentsByReviewId(req.params.review_id).then((comments) => {
+    res.status(200).send({ "comments": comments });
+  })
     .catch((err) => {
       next(err)
     })
-  }
\ No newline at end of file
+}
+
+exports.postCommentByReviewId = (req, res, next) => {
+  insertCommentByReviewId(req.body).then((comment) => {
```

Comment:
The id needs to be taken from req.params not the request body. You have doen this in your getCommentsByReviewId controller above. You'll just need to pass more arguments to your insertCommentByReviewId model when you have got hold of it and the request.